docs/tutorial.md

```diff
@@ -0,0 +1,183 @@
+# Oria Operator Tutorial
+
+### An in-depth walkthrough of building and running a oria-operator with memcached-operator.
```

Comment:
I would just make this a paragraph after the title instead of a subsection

```suggestion
An in-depth walkthrough of building and running a oria-operator with memcached-operator.
```